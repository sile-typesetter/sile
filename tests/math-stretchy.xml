<sile>

<script src="packages/math" />

Stretchy parentheses, text, MathML:
<mathml>
  <mrow>
    <mi>Γ</mi><mo>(</mo><mfrac><mi>ζ</mi><mn>2</mn></mfrac><mo>)</mo>
    <mo>+</mo> <msup><mi>x</mi><mn>2</mn></msup>
    <mrow>
      <mo>(</mo><mi>x</mi><mo>+</mo><mn>1</mn><mo>)</mo>
    </mrow>
  </mrow>
</mathml>

Stretchy parentheses, display, MathML:
<mathml mode="display">
  <mrow>
    <mi>Γ</mi><mo>(</mo><mfrac><mi>ζ</mi><mn>2</mn></mfrac><mo>)</mo>
    <mo>+</mo> <msup><mi>x</mi><mn>2</mn></msup>
    <mrow>
      <mo>(</mo><mi>x</mi><mo>+</mo><mn>1</mn><mo>)</mo>
    </mrow>
  </mrow>
</mathml>

Stretchy parentheses, text, TeX:
<math>
\Gamma(\frac{\zeta}{2}) + x^2{(x+1)}
</math>

Stretchy parentheses, display, TeX:
<math mode="display">
\Gamma(\frac{\zeta}{2}) + x^2{(x+1)}
</math>

</sile>
