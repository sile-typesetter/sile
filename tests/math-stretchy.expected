Set paper size 	595.275597	841.8897729
Begin page
Mx 	49.7638
My 	53.0855
Set font 	Gentium Plus;10;400;;normal;;LTR
T	54 87 85 72 87 70 75 92 w=35.0439	(Stretchy)
Mx 	87.4828
T	83 68 85 72 81 87 75 72 86 72 86 w=49.9072	(parentheses)
Mx 	137.3901
T	15 w=2.2900	(,)
Mx 	142.3552
T	87 72 91 87 w=16.5332	(text)
Mx 	158.8884
T	15 w=2.2900	(,)
Mx 	163.8536
T	48 68 87 75 48 47 w=34.4336	(MathML)
Mx 	198.2872
T	29 w=2.2900	(:)
Mx 	203.2523
Set font 	Libertinus Math;10;400;;normal;;LTR
T	830 w=4.9700	(nil)
Mx 	208.2223
Mx 	3.3500
T	3657 a=4.6500	(nil)
Draw line	212.8723	50.1605	3.7200	0.6500
Mx 	212.9683
My 	47.6785
Set font 	Libertinus Math;8;400;;normal;;LTR
T	864 w=3.5280	(nil)
Mx 	212.8723
My 	57.8855
T	19 w=3.7200	(nil)
Mx 	216.5923
My 	53.0855
Set font 	Libertinus Math;10;400;;normal;;LTR
Mx 	3.3200
T	3658 a=4.6500	(nil)
Mx 	223.4974
T	12 w=6.4800	(nil)
Mx 	232.2325
T	2735 w=5.4100	(nil)
Mx 	237.6425
My 	49.9855
Set font 	Libertinus Math;8;400;;normal;;LTR
T	19 w=3.7200	(nil)
Mx 	241.7925
My 	53.0855
Set font 	Libertinus Math;10;400;;normal;;LTR
T	9 w=3.5600	(nil)
Mx 	245.3525
T	2735 w=5.4100	(nil)
Mx 	253.0175
T	12 w=6.4800	(nil)
Mx 	261.7526
T	18 w=4.6500	(nil)
Mx 	266.4026
T	10 w=3.5600	(nil)
Mx 	49.7638
My 	66.4698
Set font 	Gentium Plus;10;400;;normal;;LTR
T	54 87 85 72 87 70 75 92 w=35.0439	(Stretchy)
Mx 	87.4891
T	83 68 85 72 81 87 75 72 86 72 86 w=49.9072	(parentheses)
Mx 	137.3963
T	15 w=2.2900	(,)
Mx 	142.3677
T	71 76 86 83 79 68 92 w=29.3018	(display)
Mx 	171.6695
T	15 w=2.2900	(,)
Mx 	176.6408
T	48 68 87 75 48 47 w=34.4336	(MathML)
Mx 	211.0744
T	29 w=2.2900	(:)
Mx 	263.5634
My 	93.1683
Set font 	Libertinus Math;10;400;;normal;;LTR
T	830 w=4.9700	(nil)
Mx 	268.5334
Mx 	3.6700
T	3811 a=4.9700	(nil)
Draw line	273.5034	90.2433	4.6500	0.6500
Mx 	273.6234
My 	85.9733
T	864 w=4.4100	(nil)
Mx 	273.5034
My 	100.1683
T	19 w=4.6500	(nil)
Mx 	278.1534
My 	93.1683
Mx 	3.6700
T	3819 a=4.9700	(nil)
Mx 	285.3456
T	12 w=6.4800	(nil)
Mx 	294.0478
T	2735 w=5.4100	(nil)
Mx 	299.4578
My 	89.4183
Set font 	Libertinus Math;8;400;;normal;;LTR
T	19 w=3.7200	(nil)
Mx 	303.6078
My 	93.1683
Set font 	Libertinus Math;10;400;;normal;;LTR
T	9 w=3.5600	(nil)
Mx 	307.1678
T	2735 w=5.4100	(nil)
Mx 	314.8000
T	12 w=6.4800	(nil)
Mx 	323.5022
T	18 w=4.6500	(nil)
Mx 	328.1522
T	10 w=3.5600	(nil)
Mx 	49.7638
My 	121.2613
Set font 	Gentium Plus;10;400;;normal;;LTR
T	54 87 85 72 87 70 75 92 w=35.0439	(Stretchy)
Mx 	87.4848
T	83 68 85 72 81 87 75 72 86 72 86 w=49.9072	(parentheses)
Mx 	137.3921
T	15 w=2.2900	(,)
Mx 	142.3592
T	87 72 91 87 w=16.5332	(text)
Mx 	158.8924
T	15 w=2.2900	(,)
Mx 	163.8596
T	55 72 59 w=16.2793	(TeX)
Mx 	180.1389
T	29 w=2.2900	(:)
Mx 	185.1060
Set font 	Libertinus Math;10;400;;normal;;LTR
T	830 w=4.9700	(nil)
Mx 	190.0760
Mx 	3.3500
T	3657 a=4.6500	(nil)
Draw line	194.7260	118.3363	3.7200	0.6500
Mx 	194.8220
My 	115.8543
Set font 	Libertinus Math;8;400;;normal;;LTR
T	864 w=3.5280	(nil)
Mx 	194.7260
My 	126.0613
T	19 w=3.7200	(nil)
Mx 	198.4460
My 	121.2613
Set font 	Libertinus Math;10;400;;normal;;LTR
Mx 	3.3200
T	3658 a=4.6500	(nil)
Mx 	205.3531
T	12 w=6.4800	(nil)
Mx 	214.0902
T	2735 w=5.4100	(nil)
Mx 	219.5002
My 	118.1613
Set font 	Libertinus Math;8;400;;normal;;LTR
T	19 w=3.7200	(nil)
Mx 	223.6502
My 	121.2613
Set font 	Libertinus Math;10;400;;normal;;LTR
T	9 w=3.5600	(nil)
Mx 	227.2102
T	2735 w=5.4100	(nil)
Mx 	234.8773
T	12 w=6.4800	(nil)
Mx 	243.6143
T	18 w=4.6500	(nil)
Mx 	248.2643
T	10 w=3.5600	(nil)
Mx 	49.7638
My 	134.6457
Set font 	Gentium Plus;10;400;;normal;;LTR
T	54 87 85 72 87 70 75 92 w=35.0439	(Stretchy)
Mx 	87.4911
T	83 68 85 72 81 87 75 72 86 72 86 w=49.9072	(parentheses)
Mx 	137.3983
T	15 w=2.2900	(,)
Mx 	142.3717
T	71 76 86 83 79 68 92 w=29.3018	(display)
Mx 	171.6734
T	15 w=2.2900	(,)
Mx 	176.6468
T	55 72 59 w=16.2793	(TeX)
Mx 	192.9261
T	29 w=2.2900	(:)
Mx 	263.5634
My 	161.3441
Set font 	Libertinus Math;10;400;;normal;;LTR
T	830 w=4.9700	(nil)
Mx 	268.5334
Mx 	3.6700
T	3811 a=4.9700	(nil)
Draw line	273.5034	158.4191	4.6500	0.6500
Mx 	273.6234
My 	154.1491
T	864 w=4.4100	(nil)
Mx 	273.5034
My 	168.3441
T	19 w=4.6500	(nil)
Mx 	278.1534
My 	161.3441
Mx 	3.6700
T	3819 a=4.9700	(nil)
Mx 	285.3456
T	12 w=6.4800	(nil)
Mx 	294.0478
T	2735 w=5.4100	(nil)
Mx 	299.4578
My 	157.5941
Set font 	Libertinus Math;8;400;;normal;;LTR
T	19 w=3.7200	(nil)
Mx 	303.6078
My 	161.3441
Set font 	Libertinus Math;10;400;;normal;;LTR
T	9 w=3.5600	(nil)
Mx 	307.1678
T	2735 w=5.4100	(nil)
Mx 	314.8000
T	12 w=6.4800	(nil)
Mx 	323.5022
T	18 w=4.6500	(nil)
Mx 	328.1522
T	10 w=3.5600	(nil)
Mx 	295.2916
My 	780.6177
Set font 	Gentium Plus;10;400;;normal;;LTR
T	20 w=4.6924	(1)
End page
Finish
